src/styles/mixins/transition.scss

```diff
@@ -0,0 +1,3 @@
+@mixin transition($property, $duration: $transition) {
+  transition: $property $duration;
+}
```

Comment:
it could just be `mixins.scss`
and contain multiple mixins - if you had 5+ mixing you wouldn't create a file for each right?